source/unified-test-format/unified-test-format.rst

```diff
@@ -462,6 +477,78 @@ The structure of this object is as follows:
+       
+          - ``name``: the name of the event, i.e. ``CommandStartedEvent``.
+          - ``commandName``: the name of the command, e.g. ``insert``.
+          - ``startTime``: the (floating-point) number of seconds since
+            the Unix epoch when the command began executing.
+          - ``address``: the address of the server to which the command
+             was sent, e.g. ``localhost:27017``.
+        
+        - For ``CommandSucceededEvent``:
+       
+          - ``name``: the name of the event, i.e. ``CommandSucceededEvent``.
+          - ``commandName``: the name of the command, e.g. ``insert``.
+          - ``duration``: the time, in (floating-point) seconds, it took for
+            the command to execute.
+          - ``startTime``: the (floating-point) number of seconds since
+            the Unix epoch when the command began executing.
+          - ``address``: the address of the server to which the command
+             was sent, e.g. ``localhost:27017``.
+        
+        - For ``CommandFailedEvent``:
+       
+          - ``name``: the name of the event, i.e. ``CommandFailedEvent``.
+          - ``commandName``: the name of the command, e.g. ``insert``.
+          - ``duration``: the time, in (floating-point) seconds, it took for
+            the command to execute.
+          - ``failure``: this field MUST contain a textual description
+             of the error encountered while executing the command.
+          - ``startTime``: the (floating-point) number of seconds since
+            the Unix epoch when the command began executing.
+          - ``address``: the address of the server to which the command
+             was sent, e.g. ``localhost:27017``.
+             
+        - For connection pool events (``PoolCreatedEvent``,
+          ``PoolReadyEvent``, ``PoolClearedEvent``, ``PoolClosedEvent``):
+               
+          - ``name``: the name of the event, e.g. ``PoolCreatedEvent``.
+          - ``time``: the (floating-point) number of seconds since the
+            Unix epoch when the event was published.
+          - ``address``: the address of the server that the command was
+            published for, e.g. ``localhost:27017``.
+        
+        - For connnection events (``ConnectionCreatedEvent``,
+          ``ConnectionReadyEvent``, ``ConnectionClosedEvent``,
+          ``ConnectionCheckOutStartedEvent``, ``ConnectionCheckOutFailedEvent``,
+          ``ConnectionCheckedOutEvent``, ``ConnectionCheckedInEvent``):
+               
+          - ``name``: the name of the event, e.g. ``ConnectionCreatedEvent``.
+          - ``time``: the (floating-point) number of seconds since the
+            Unix epoch when the event was published.
+          - ``address``: the address of the server that the command was
+            published for, e.g. ``localhost:27017``.
+          - ``connectionId``: the identifier for the connection for which
+            the event was published.
```

Comment:
Looking at the history of command monitoring spec, references to connenctionId go back to 2015 when the spec was written hence it seems to me they aren't specifically referencing the same connection id that CMAP uses, necessarily.

I imagine it would be sensible for this connection id to be the same as the CMAP ones, but this doesn't seem to be strictly required.

Reply:
I created https://jira.mongodb.org/browse/DRIVERS-1548 for this.